### DISCUSSION TITLE
Pid Files

### DISCUSSION THREAD

**Discussion by GitHed** [kohsuke/winsw#819] [Q&A]
Can you explain the pid files?

They seem to fail if they don't exist, which apparently makes the first start a bit difficult. It apparently doesn't create the file, which seems odd, and if the file does exist, if it doesn't have data that also causes it to fail to start.

Invalid PID file number in 'C:\JenkinsAgent\jenkins-agent.pid'. The runaway process won't be checked
System.FormatException: Input string was not in a correct format.

The requested PID file 'C:\JenkinsAgent\jenkins-agent.pid' does not exist. The runaway process won't be checked

I guess the actual issue here is the error message of cannot find the file, which doesn't actually specify which file it cannot find, and in my cause it was a path issue with the executable.

**Comment by And9686** [comment]
I'm got the same message: " The requested PID file 'C:\tools\Jenkins\jenkins.pid' does not exist. The runaway process won't be checked"
Is this problem soved to you? I didn't understand the path issue? I followed exactly the tutorial of the Jenkins old man so I don't really know how to get this to work.
